tests/test_physical_systems/test_converters.py

```diff
@@ -1129,38 +1069,37 @@ def test_registered(self):
         dummy_converters[2].action_space = Box(0, 1, shape=(1,))
         conv = gem.utils.instantiate(cv.PowerElectronicConverter, self.key, subconverters=dummy_converters)
         assert type(conv) == self.class_to_test
-        assert conv._subconverters == dummy_converters
+        assert conv._sub_converters == dummy_converters
 
-    @pytest.mark.parametrize("tau, dead_time, interlocking_time, kwargs", [
-        (1, True, 0.1, {'subconverters': ['Cont-1QC', 'Cont-B6C', 'Cont-4QC']}),
-        (0.1, False, 0.0, {'subconverters': ['Cont-1QC', 'Cont-B6C', 'Cont-4QC']}),
+    @pytest.mark.parametrize("tau, interlocking_time, kwargs", [
+        (1, 0.1, {'subconverters': ['Cont-1QC', 'Cont-B6C', 'Cont-4QC']}),

```

Comment:
subconverters here seem to miss the underscore

Reply:
Yes and no. The term subconverters / sub_converters is used inconistently within GEM. I already work on that topic in a different branch (phyiscal_system_interface_update). In my opinion "sub_converters" would be gramatically correct. At this point, the argument is expected as "subconverters".